proto/wallirpc.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+import "google/api/annotations.proto";
+
+package wallirpc;
+
+service Walli {
+  rpc getInfo (GetInfoRequest) returns (GetInfoResponse);
+}
+
+message GetInfoRequest {}
+message GetInfoResponse {
+   int32 version = 1;
```

Comment:
Are the `[json_name = "version"]` in the proto file of XUD just used for the webproxy or is there a different reason for which you don't use them here?

Reply:
I didn't think we need it because we don't have a web proxy yet.